src/cheri/zysentry.adoc

```diff
@@ -0,0 +1,67 @@
+[#section_zysentry,reftext="{cheri_sentry_ext_name}"]
+== "{cheri_sentry_ext_name}" Extension for CHERI Sentry Capabilities
+
+[#sentry_cap,reftext="sentry capability"]
+=== Sentry Capabilities
+
```

Comment:
I think this requires defining another extension to add a new backward jump type to keep the semantics of return sentries simple (as discussed in https://github.com/riscv/riscv-cheri/issues/672